cli/command/swarm/leave.go

```diff
@@ -19,15 +19,15 @@ func newLeaveCommand(dockerCli *command.DockerCli) *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "leave [OPTIONS]",
-		Short: "Leave a swarm",
+		Short: "Leave a swarm. If the node is a manager, demote it to a worker before removing it from the swarm.",
```

Comment:
I'm a little concerned that this could be misleading, because it makes it sound like the command will automatically demote a worker. I think it's best to leave this as `Leave a swarm`, since other commands' short descriptions are short phrases.